src/lib/hooks/use-best-quote/utils/flashmint.ts

```diff
@@ -86,19 +86,11 @@ async function getEnhancedFlashMintQuote(
 
       const inputTokenAmountUsd =
         parseFloat(
-          formatAmountFromWei(
-            inputTokenAmount.toBigInt(),
-            inputToken.decimals,
-            10,
-          ),
+          formatWei(inputTokenAmount.toBigInt(), inputToken.decimals),
```

Comment:
Those still had 10 digit precision, so I guess it doesn't matter if they theoretically could get bigger

Reply:
I don't believe it mattered here since we're multiplying it by `inputTokenPrice`, which would change the number of decimals on the value